services/src/main/java/org/jboss/windup/web/services/validators/NotBlankConstraint.java

```diff
@@ -0,0 +1,22 @@
+package org.jboss.windup.web.services.validators;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import javax.validation.Constraint;
+import javax.validation.Payload;
+
+/**
+ * @author <a href="mailto:zizka@seznam.cz">Ondrej Zizka</a>
+ */
+@Constraint(validatedBy = NotBlankValidator.class)
+@Target({ ElementType.FIELD, ElementType.PARAMETER })
+@Retention(RetentionPolicy.RUNTIME)
+public @interface NotBlankConstraint
```

Comment:
I would, but that gives a message, at best, like "Foo must be not empty or at most 4096 characters" which is kind of weird to combine like that.